lib/workers/global/autodiscover.ts

```diff
@@ -13,6 +13,15 @@ function repoName(value: string | { repository: string }): string {
 export async function autodiscoverRepositories(
   config: AllConfig
 ): Promise<AllConfig> {
+  if (config.platform === 'local') {
+    if (config.repositories?.length) {
+      logger.warn(
+        { repositories: config.repositories },
+        'platform=local supports only cwd'
+      );
+    }
+    config.repositories = ['local'];
+  }
```

Comment:
In theory we could support multiple repos in a single run, but it adds complexity and I'm not sure if it's really a valid or common use case?

Reply:
i think a single repo is ok for now. maybe throw or log as error?

Reply:
Maybe let's abort/error if any arguments are given

Reply:
sounds good